views/update/review-update-statement.html

```diff
@@ -127,10 +175,34 @@ <h1 class="govuk-heading-xl">Review the information before submitting this updat
                 "nochange-changelink-entity-review-statement-button"
               ) ]
             }
-          }
+          } if not isRemoveJourney
         ]
     }) }}
+    <br>
 
+    {% if isRemoveJourney %}
+      <h2 class="govuk-heading-l">Changes to the overseas entity</h2>
+      {{ govukSummaryList({
+        rows: [
+          {
+            key: {
+              text: "Do you need to make any changes to this overseas entity?"
+            },
+            value: {
+              text: noChange
+            },
+            attributes: {
+              "data-event-id": "nochange-changelink-entity-review-statement-button"
+            }
+          }
+        ]
+      }) }}
+      <br>
+    {% endif %}
+
+    {% if isRemoveJourney %}
+      {% set hidePresenterChangeLink = true %}
```

Comment:
Just wondering why you did this, rather than let presenter.html test against the `isRemoveJourney` flag directly?

Reply:
There might be other situations where change links might not want to be shown so figured it more relevant to have a hidePresenterChangeLink value. As the presenter.html is a re-usable 'chunk' , by having a hidePresenterChangeLink, we can let whatever uses it decide if change links to be shown or not.